ivy/functional/backends/torch/elementwise.py

```diff
@@ -486,7 +486,10 @@ def abs(
 
 
 def logaddexp(
-    x1: torch.Tensor, x2: torch.Tensor, /, *, out: Optional[torch.Tensor] = None
```

Comment:
Please revert the change made here with the parameters.

Reply:
reverted the change